doc/source/examples/unresolved-cfd-dem/boycott-effect/boycott-effect.rst

```diff
@@ -89,9 +89,11 @@ The section on model parameters is explained in the DEM examples. We show the ch
 .. code-block:: text
 
     subsection model parameters
-      set contact detection method               = dynamic
-      set contact detection frequency            = 10
-      set neighborhood threshold                 = 1.3
+      subsection contact detection
+        set contact detection method = dynamic
+        set neighborhood threshold   = 1.3
+        set frequency                = 10
```

Comment:
```suggestion
```
I don't suggest using a frequency of 10 here, this might cause mis-synchronization. 10 is high for frequency for dynamic contact search.